doc/api/next_api_changes/behavior/27408-SR.rst

```diff
@@ -0,0 +1,8 @@
+Classes derived from Axis must implement get_tick_params
```

Comment:
Wouldn't it be sufficient to implement the translation, either as a dict or as `_translate_tick_params()`?

Reply:
Sure, but `_translate_tick_params` is 18 LOC (not counting continuation lines) whereas `get_tick_params` is just 2.  

Another thing is that set/get_tick_params currently doesn't work correctly for Axes3D (I'm going to open an issue about it) and I assume that it's easier fix with dedicated setters/getters (didn't look into it closely yet)

Reply:
On second thought, I think you're right and it's less convoluted to make the translation dicts axis-specific in the first place rather than modify a generic dict. I'll change it later this week, changing this PR to draft in the meantime.